xcube/webapi/res/openapi.yml

```diff
@@ -626,6 +650,20 @@ components:
       required: true
       schema:
         type: string
+    statstype:
+      name: type
+      in: path
+      description: Type of Summary Statistics
+      required: true
+      schema:
+        type: string
+    geometry:
+      name: geometry
+      in: path
+      description: Geometry for Filtering a Dataset
+      required: true
+      schema:
+        type: string
```

Comment:
What does this have to do with #142? 
Where does the requirement come from?

Reply:
These changes were ideas related to dev work related to the [Xcube viewer issue 50](https://github.com/dcs4cop/xcube-viewer/issues/50) based on discussions with kerstin. No active development.